src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [inputData, setFilteredList] = useState('');
```

Comment:
It's beneficial to use consistent naming conventions to make the code more understandable.
```suggestion
  const [query, setQuery] = useState('');
```